frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -88,6 +92,14 @@ const Metric = (props: MetricProps) => {
         const mode = (props.type === "none") ? [] : ["gauge"];
         showValue && mode.push("number");
         (delta !== undefined) && mode.push("delta");
+        const deltaIncreasing = positiveDeltaColor?{
+            symbol: "▲",
```

Comment:
Do you have to specify the symbol ?

Reply:
I got Typescript error if I didn't specify it.